project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/controllers/activity.js

```diff
@@ -188,6 +191,8 @@ angular.module('c4mApp')
             // The "Count" variable will go down as we are filtering with the lowest activity Timestamp.
             $scope.showMoreButton = data.data.length >= $scope.range;
           }
+            // Removes loading state class
```

Comment:
@guytepper 
the test failed because you're missing a `.` at the end of the comment.